src/site/content/en/blog/coop-coep/index.md

```diff
@@ -173,9 +173,9 @@ powerful features like `performance.measureMemory()`.
 
 {% Aside 'caution' %}
 The
-[`self.crossOriginIsolated`](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/crossOriginIsolated.)
-property is [still under development and not available yet in Chrome as of
-version 83](https://www.chromestatus.com/feature/5953286387531776).
```

Comment:
@agektmr should we replace the 404ing `chromestatus.com` link with something else?

Reply:
Actually I'm going to approve/merge this PR because I don't want to keep all of these broken links around just to sort out what to do with one of them.

Reply:
LGTM. Thanks!